drivers/sensor/paw3212/paw3212.c

```diff
@@ -548,7 +548,7 @@ static void trigger_handler(struct k_work *work)
 	sensor_trigger_handler_t handler;
 	int err = 0;
 	struct paw3212_data *data = CONTAINER_OF(work, struct paw3212_data,
-						     trigger_handler_work);
+						 trigger_handler_work);
```

Comment:
@MarekPieta

Reply:
There is no need for such change. `trigger_handler_work` is a `k_work` (not `k_work_delayable`) so that type of `work` pointer already matches it.
For delayable works, a regular work item (that is member of delayable work structure) is passed as work handler argument, so we need to add `.work` to access it.